packages/webapp/src/_app/hooks/queries.ts

```diff
@@ -213,13 +230,20 @@ export const useHeadDelegate = () =>
         ...queryHeadDelegate,
     });
 
-export const useParticipantsInCompletedRound = (
-    electionRound?: number,
-    member?: EdenMember
-) => {
+export const useParticipantsInCompletedRound = (electionRound?: number) => {
+    const { data: member } = useCurrentMember();
+    const { data: voteData } = useVoteDataRow(member?.account);
+    console.info("voteData:", voteData);
+
+    const isStillParticipating = Boolean(voteData);
+    console.info(`isStillParticipating[${isStillParticipating}]`);
```

Comment:
We use `isStillParticipating` as a way to see whether you're the winner of the previous round or not, but what if the election is ongoing and in Round 3? You won Round 1 and lost Round 2. And you want to look at Round 1 results. You're waiting for the election to end and observing. We would be unable to show you that you won Round 1, wouldn't we? (Maybe we have to compare Round 1 with Round 2 to find the common participant to show you that you're the winner of Round 1?)

@cool-ant

Reply:
Also, if you're not signed in as the winner, the winner is getting omitted from the list entirely.